README.md

```diff
@@ -151,3 +151,7 @@ Below you can find information on how to push local code changes to a kind clust
 - push image to local registry: `podman push localhost:5000/maiqueb/multus-dynamic-networks-controller`
 - update manifests to use the generated image: `IMAGE_REGISTRY=localhost:5000/maiqueb make manifests`
 - deploy the controller: `kubectl apply -f manifests/dynamic-networks-controller.yaml`
+
+## Known limitations
+- the networks being hot-plugged **must** feature the `name` attribute in the `spec.config`. This is tracked in this [issue](https://github.com/maiqueb/multus-dynamic-networks-controller/issues/45).
+- the pod controller is not level driven. This is tracked in this [RFE](https://github.com/maiqueb/multus-dynamic-networks-controller/issues/48).
```

Comment:
Is "level driven" a well known (kubernetes) lingo?

Reply:
hmm. No. This predates all that; its a term from circuits - edge trigger vs level trigger.

Reply:
Oh you are such a nerd! I'm familiar with edge vs level there, but I would never make the connection in this README

Reply:
Does the provided stack overflow answer help ? ... 

I am open to any other suggestions @phoracek

Reply:
perhaps "hot-plug requests done while the controller was offline won't be acted upon (or reconciled) once the controller comes back online"?

Reply:
whoops. /me facepalms